src/main/java/work/xeltica/craft/core/utils/DiscordService.java

```diff
@@ -0,0 +1,55 @@
+package work.xeltica.craft.core.utils;
+
+import github.scarsz.discordsrv.DiscordSRV;
+import github.scarsz.discordsrv.dependencies.jda.api.entities.Member;
+import github.scarsz.discordsrv.dependencies.jda.api.entities.TextChannel;
+import net.luckperms.api.LuckPerms;
+import org.bukkit.Bukkit;
+import org.bukkit.entity.Player;
+
+/**
+ * @author raink1208
+ */
+public class DiscordService {
+    public DiscordService() {
+        instance = this;
+    }
+
+    public static DiscordService getInstance() {
+        return instance;
+    }
+
+    public Member getMember(Player player) {
+        if (!linkedToDiscord(player)) return null;
+
+        final String discordId = DiscordSRV.getPlugin().getAccountLinkManager().getDiscordId(player.getUniqueId());
+        return DiscordSRV.getPlugin().getMainGuild().getMemberById(discordId);
+    }
+
+    public void reportDiscord(Player badPlayer, String abuses, String time, String command) {
+        final var guild = DiscordSRV.getPlugin().getMainGuild();
+        final var channel = guild.getGuildChannelById(reportChannelID);
+        if (channel instanceof TextChannel textChannel) {
+            textChannel.sendMessage(String.format(reportTemplate, badPlayer.getName(), abuses, time+command)).queue();
```

Comment:
ガイドラインを特に決めていない中申し訳ないのですが、演算子の前後空白は次の様な感じでお願いします

```suggestion
            textChannel.sendMessage(String.format(reportTemplate, badPlayer.getName(), abuses, time + command)).queue();
```